.github/workflows/gh-actions.yml

```diff
@@ -30,7 +30,9 @@ jobs:
         with:
           python-version: '2.7' # needed by jerry-debugger
       - run: $RUNNER -q --jerry-tests
+      - run: $RUNNER -q --jerry-tests --build-debug
```

Comment:
Now each job takes almost 2 times longer, since it's running the tests in release and debug mode.

I'd suggest to create  <original_job_name>_Debug` jobs instead of burden each job with running it in debug mode as well. GH action can run 20 jobs in parallel so take it as an advantage.

Reply:
https://github.com/jerryscript-project/jerryscript/runs/1832072314?check_suite_focus=true
This is the test result, which test need split?
Or all need to be split?

Reply:
https://github.com/jerryscript-project/jerryscript/runs/1832072314?check_suite_focus=true This already consume 12minutes.

Reply:
Well, most of them are okay within one job, but I'd definitely split these:
![image](https://user-images.githubusercontent.com/22135179/106920194-24861400-670b-11eb-9cd9-976d1f261319.png)

These seems the biggest slowdowns.

Reply:
https://github.com/jerryscript-project/jerryscript/pull/4389 fixes ES5_1, others OK to me